framework-api-unifiedsearch/src/main/java/uk/gov/justice/services/unifiedsearch/UnifiedSearchIndexer.java

```diff
@@ -1,7 +1,9 @@
 package uk.gov.justice.services.unifiedsearch;
 
-import javax.json.JsonObject;
+import uk.gov.justice.services.messaging.JsonEnvelope;
+
+import javax.enterprise.event.Event;
 
 public interface UnifiedSearchIndexer {
-    void indexData(final JsonObject index);
+    void indexData(final Event<JsonEnvelope> eventWithJoltTransformedPayload);
```

Comment:
should be...
void indexData(final Envelope<JsonObject> eventWithJoltTransformedPayload);